src/Components/Dashboard/MainPage.js

```diff
@@ -1,14 +1,71 @@
-import React from 'react'
-import PageLayout from "../Pages/PageLayout"
+import React, { useEffect, useState } from 'react';
+import PageLayout from '../Pages/PageLayout';
+import { useDispatch } from 'react-redux';
+import { connect } from 'react-redux';
+import { getFeedDetails } from '../../Auth/Actions/feedActions';
+import './DashBoard.css';
+import Unavailiabe from '../../Utils/unavailiable1.png';
+import Feeds from './Feeds';
 
-const MainPage = () => {
+const MainPage = ({ feeds, user }) => {
+    const [offSet, setOffSet] = useState(0);
+    const feedsLength = feeds?.length;
+    const dispatch = useDispatch();
+    const [isLoading, setIsLoading] = useState(false);
+    useEffect(() => {
+        window.scrollTo({
+            top: 0,
+            left: 0,
+            behavior: 'smooth',
+        });
+        dispatch(getFeedDetails(offSet, setIsLoading));
```

Comment:
why is this still here? get the loading state from redux store now

Reply:
Ok sir. I have now been able to get the loading state from redux. And i have implemented it in my code